NSS-Day-Cohort-42/rare-API-media-moguls
# Cb profile images

#### Changes Made
1. Modified auth.py in views directory to allow for user to send picture from client side, then have it stored in media/images
2. Modified the Rareuser model to allow pictures to be uploaded to a specific file, and for user to have option to not have a profile image
​
#### Steps to Review
1. Checkout this branch locally.
```
git fetch --all
git checkout cb-profile-images
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> When User follows all of the client side steps (as detailed in client side PR where user registers an account and submits a profile picture), when that user submits a picture it should be stored in the media/images folder
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
